ast.go

```diff
@@ -1058,7 +1094,22 @@ func (node *ColumnDef) String() string {
 		}
 		constraint = " " + strings.Join(constraints, " ")
 	}
-	return fmt.Sprintf("%s %s%s", node.Name, node.Type, constraint)
+	return fmt.Sprintf("%s %s%s", node.Column, node.Type, constraint)
+}
+
+// HasPrimaryKey checks if column definition has a primary key constraint.
+func (node *ColumnDef) HasPrimaryKey() bool {
+	if len(node.Constraints) == 0 {
+		return false
+	}
+
+	for _, constraint := range node.Constraints {
```

Comment:
Go through all constraints and check if there is a primary key inside it. Remembering that a constraint in this case is an `ColumnConstraint` interface and the `*ColumnConstraintPrimaryKey` implements that interface.